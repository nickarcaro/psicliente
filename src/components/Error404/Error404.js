import React from "react";

const Error404 = () => {
  return <div>error404notfound</div>;
};
export default Error404;
