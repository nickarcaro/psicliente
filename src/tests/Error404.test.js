import React from "react";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";
import Error404 from "../pages/Error404";

describe("Pruebas de <Error404/>", () => {
  test("debe mostrar <Error404/> correctamente", () => {
    const texto = "Error 404 not found";

    const wrapper = shallow(<Error404 />);
    expect(wrapper).toMatchSnapshot();

    const textoDiv = wrapper.find("div").text();
    expect(textoDiv).toBe(texto);
  });
});
