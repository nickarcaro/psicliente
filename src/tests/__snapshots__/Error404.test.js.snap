// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de <Error404/> debe mostrar <Error404/> correctamente 1`] = `
<div>
  Error 404 not found
</div>
`;
